NSS-Day-Cohort-55/tabloidfullstack-adequately-powerful-ibexes
# Wm view users posts

# Description
Allows the user to click on "My Posts" from the home menu and view their own posts. Closes ticket Number #44
Fixes # (issue)


# Testing Instructions
Login as rdo1@timesonline.co.ukx with your password. Click on "My Posts" and you will see a list of the current users post. Database Default is one post for this user.
